### PR TITLE
[IMP] mail: added test for no suggested recipient in lognote

### PR DESCRIPTION
**Pull request by pka-odoo**
Added test for no suggested recipient in log note

Task - 2371744

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/mail/static/src/components/chatter/chatter_suggested_recipient_tests.js [hunk 1]
```diff
@@ -374,6 +374,34 @@ QUnit.test("suggested recipients list display 3 suggested recipient and 'show mo
...
+    });
+    await this.createChatterComponent({ chatter });
+    await afterNextRender(() =>
+        document.querySelector(`.o_ChatterTopbar_buttonLogNote`).click()
+    );
+    assert.containsNone(
+        document.body,
+        '.o_ComposerSuggestedRecipientList',
+        "Should not contain recipients when opening the composer from 'log note' button"
+    );
```
Comment (seb-odoo) [inline addons/mail/static/src/components/chatter/chatter_suggested_recipient_tests.js:402]: The goal here is not to test the interface (which was already correct before the mentioned PR), but to test that the RPC done when sending the note does not contain the recipients as param.

**Review by seb-odoo — CHANGES_REQUESTED** [review]

**Comment by pka-odoo** [comment]
@seb-odoo Changes are done. Thanks

File: addons/mail/static/src/components/chatter/chatter_suggested_recipient_tests.js [hunk 2]
```diff
@@ -374,6 +374,42 @@ QUnit.test("suggested recipients list display 3 suggested recipient and 'show mo
...
+    });
+    this.data['res.fake'].records.push({
+        id: 10,
+        partner_ids: [100],
+    });
+    await this.start({
+        async mockRPC(route, args) {
+            if (args.model === 'res.fake' && args.method === 'message_post') {
+                assert.notOk(args.kwargs.partner_ids.length,
+                    'message_post should not contain partners for lognote');
```
Comment (seb-odoo) [inline addons/mail/static/src/components/chatter/chatter_suggested_recipient_tests.js:393]:
```suggestion
                assert.strictEqual(
                    args.kwargs.partner_ids.length,
                    0,
                    "message_post should not contain suggested recipients when posting an internal note"
                );
```

**Review by seb-odoo — CHANGES_REQUESTED** [review]

**Comment by pka-odoo** [comment]
@seb-odoo Done.

File: addons/mail/static/src/components/chatter/chatter_suggested_recipient_tests.js [hunk 3]
```diff
@@ -374,6 +374,45 @@ QUnit.test("suggested recipients list display 3 suggested recipient and 'show mo
...
+        id: 10,
+        partner_ids: [100],
+    });
+    await this.start({
+        async mockRPC(route, args) {
+            if (args.model === 'res.fake' && args.method === 'message_post') {
+                assert.strictEqual(
+                    args.kwargs.partner_ids.length,
+                    0,
+                    'message_post should not contain partners for lognote'
```
Comment (seb-odoo) [inline addons/mail/static/src/components/chatter/chatter_suggested_recipient_tests.js:395]: My suggestion also included this text:

"message_post should not contain suggested recipients when posting an internal note"

**Review by seb-odoo — COMMENTED** [review]
Please squash commits with [FIX] tag after comment below

**Review by seb-odoo — APPROVED** [review]
Thank you

@robodoo r+

**Pull request closed without merging**